src/actions/marketing-actions.js

```diff
@@ -211,10 +220,10 @@ export const saveMarketingSetting =
 
     if (entity.id) {
       if (file)
-        return putFile(
+        return postFile(
```

Comment:
this solution is not correct
u dont need to change this
u need to reset the entity.id to zero when its deleted on remove